modules/introduction/partials/new-features-80.adoc

```diff
@@ -62,6 +62,45 @@ curl --get -u <username:password> \
 [#section-new-feature-data-service]
 === Data Service
 
+[#mb-62777]
+https://jira.issues.couchbase.com/browse/MB-62777[MB-62777] Magma with 128 vBuckets is now the default storage engine::
+In earlier versions of Couchbase Server, if you did not specify which storage engine to use when creating a new bucket, it would use Couchstore.
+Once you upgrade your entire cluster to Couchbase Server Enterprise 8.0, the default storage engine for new buckets is Magma with 128 vBuckets.
+
++
+Magma buckets with 128 vBuckets is a new feature that provides the advantages of the Magma storage engine while reducing the minimum memory quota. 
+Before Couchbase Server 8.0, Magma only supported using 1024 vBuckets and had a minimum memory quota of 1{nbsp}GiB per node.
+Buckets that use Magma with 128 vBuckets have a minimum memory quota of 100{nbsp}MiB per node, which is the same as Couchstore.
+
++
+NOTE: These changes do not affect Couchbase Server Community Edition because it does not support the Magma storage engine.
+
++
+This change in the default storage engine does not affect existing buckets.
+You can still create buckets that use the Couchstore storage engine or the Magma st5orage engine with 1024 vBuckets by explicitly specifying them when you create the bucket.
+
++
+See xref:learn:buckets-memory-and-storage/storage-engines.adoc[] for more information about storage engines.
+
++
+.default behavior change
+[IMPORTANT] 
+====
+The new default storage backend for buckets is a behavior change that may cause issues if you rely on the previous behavior.
+
+Before upgrading to Couchbase Server 8.0, consider the following:
+
+* If you have deployment scripts that create buckets without specifying the storage engine, those scripts  create Magma buckets with 128 vBuckets instead of Couchstore buckets after the upgrade.
+This may affect your deployment if you depend on buckets using the Couchstore storage engine.
```

Comment:
There should be another bullet item -- this is a breaking change like the first bullet item:

* If you have deployment scripts that create Magma buckets, those scripts will create Magma with 128 vBuckets instead of Magma with 1024 vBuckets.  To continue to create the same Magma buckets with 1024 vBuckets in 8.0 and later, you must specify the number of vBuckets when creating the bucket.   To set this value, use the `numVbuckets` parameter in xref:rest-api:rest-bucket-create.adoc#numvbuckets[the REST API] or `--num-vbuckets` in the xref:cli:cbcli/couchbase-cli-bucket-create.adoc[`couchbase-cli` command line tool].  If you can devote at least 1 GiB of memory to the Magma bucket, you should create a Magma bucket with 1024 vBuckets, as you will get better performance at scale.